### ISSUE TITLE
ARM11 data abort when trying to access "System Settings"

### ISSUE THREAD

**Issue by LuzPoder** [LumaTeam/Luma3DS#1230]
**System model: Old 2DS**

**SysNAND version: Probably around ver.11.8, but the specific nature of the issue does not allow to see the versions.**

**Entrypoint: Boot9Strap ver.1.3**

**Luma3DS version: 9.1**

**Luma3DS configuration/options:**

Default EmuNAND: ( )
Screen brightness: ( )
Splash: ( )
PIN lock: ( )
--
Autoboot EmuNAND: ( )
Use EmuNAND FIRM if booting with R: ( )
Enable loading external FIRMs and modules: ( )
Enable game patching: ( )
Show NAND or user string in System Settings: ( )
Show GBA boot screen in patched AGB_FIRM: ( )
Patch ARM9 access: ( )
Set developer UNITINFO: ( )
Disable ARM11 exception handlers: ( )
--

**Explanation of the issue:**
This console has got an owner different from me, which once made the AuroraNAND programming before I put the Luma + B9S with the Magnethax method. Here is a screenshot of the SD backup for clearer understanding:
![First backup](https://user-images.githubusercontent.com/11376806/54469222-93fc5e80-4795-11e9-8693-ae9d84912e12.png)
After this, I made the proper changes, applied the latest versions of Luma and B9S.
I also used the latest's Godmode9 to use GM9Megascript -> Scripts from Plailect’s Guide -> Setup Luma3DS to CTRNAND

After all my work, the SD card looks like this:
![Second backup](https://user-images.githubusercontent.com/11376806/54469301-957a5680-4796-11e9-91b4-cea23aa8da10.png)

The console works exactly as expected from Luma, runs any game and has no issues other than the one mentioned here: when it comes the time to access "System Settings", you will find the next screen (also happened before Luma + B9S):

![Error_1](https://user-images.githubusercontent.com/11376806/54469384-87790580-4797-11e9-9395-d31b29926681.png)
![Error_2](https://user-images.githubusercontent.com/11376806/54469385-8a73f600-4797-11e9-8e82-6f73e12949e1.png)

**Steps to reproduce:**

1. Do the process explained above.

2. Switch the console on.

3. Press A button to access "System Settings".

**Dump file:**
[crash_dump_00000002.zip](https://github.com/AuroraWright/Luma3DS/files/2973366/crash_dump_00000002.zip)

**Comment by TuxSH** [comment]
Can't reproduce. Try to disable arm11 exceptions handlers, maybe that'll work.

> also happened before Luma + B9S

so that's not a luma issue

**Comment by LuzPoder** [comment]
@TuxSH I did disable arm11 exceptions handlers, didn't work. Then I posted here.

> so that's not a luma issue

It is true that the error's been there from AuroraNAND, but it is also true that Luma recognizes the error and makes a dump file from it, there must be a solution!

PD: I talked with the owner, he said: "When I made the process, there was a step that totally disabled the use of internet so Nintendo would not ban the console".

Maybe that's related to the disabled System Settings?
